median/aggregated_attribute_observation.go

```diff
@@ -33,6 +34,11 @@ func aggregate(observations []median.ParsedAttributedObservation) *aggregatedAtt
 	})
 	aggregated.JuelsPerFeeCoin = observations[n/2].JuelsPerFeeCoin
 
+	slices.SortFunc(observations, func(a, b median.ParsedAttributedObservation) int {
```

Comment:
extracts the GasPriceSubunits to the aggregated observation. If it uses the default codec or codecs that do not know about this field, then field is ignored (see https://chainlink-core.slack.com/archives/C03EZMT3XPX/p1715627908908079?thread_ts=1711719895.085089&cid=C03EZMT3XPX). 

If it's on Starknet, the Starknet codec will utilize it